packages/ui-components/src/components/layout/Table.vue

```diff
@@ -118,7 +118,7 @@ const paddingRightStyle = computed(() => {
 
 const rowsWrapperClasses = computed(() => {
   const classParts = [
-    'relative grid grid-cols-12 items-center gap-6 px-4 py-1 min-w-[900px] bg-foundation'
```

Comment:
@benjaminvo  This changes the base Table element, meaning all tables everywhere will have this change. Have you validated that this reduced width doesn't break any Table layouts?

Reply:
Yep, from the initial PR description:

> I've checked other tables in the product after this change and it doesn't introduce any problems.